examples/resgen/BUILD

```diff
@@ -0,0 +1,21 @@
+load(
+    "@d2l_rules_csharp//csharp:defs.bzl",
+    "csharp_binary",
+    "csharp_resx"
+)
+
+# bazel run //examples:hello
+csharp_binary(
+    name = "hello",
+    srcs = ["hello.cs", "Strings.Designer.cs"],
+    resources = [":hello.resources"],
+    target_frameworks = ["net472"],
+    deps = ["@net//:System.Linq"],
+)
+
+csharp_resx(
```

Comment:
Looks good. If `src` -> `srcs` then it'd be idiomatic to name this rule `csharp_resx_library`.